source/_components/streamlabswater.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: page
+title: "Streamlabs Water"
+description: "Instructions on how to integrate Streamlabs Water devices with Home Assistant."
+date: 2019-02-25 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: streamlabswater.png
+ha_category:
+  - Binary Sensor
+  - Sensor
+ha_release: 0.90
```

Comment:
```suggestion
ha_release: "0.90"
```

Reply:
Can you comment on why you think this change is needed?  The vast majority of components do not quote the release version.

Reply:
Ah, perhaps because it is a numerical value otherwise and drops the trailing zero?